client/tos-modal/style.scss

```diff
@@ -0,0 +1,26 @@
+.woocommerce-payments__tos-modal {
```

Comment:
CSS looks good, but it seems like some responsive styles/fixes are needed:

<img width="466" alt="image" src="https://user-images.githubusercontent.com/5311119/96972191-d6178580-151e-11eb-87a4-ed54922fb276.png">

This is a mobile screenshot, but tablets look a bit quirkier.

Reply:
Nice catch! Fixed in https://github.com/Automattic/woocommerce-payments/pull/983/commits/7ecb8ac203af7e007eb04fc975a3348f7ce84fd3.